.github/workflows/maven.yml

```diff
@@ -5,9 +5,8 @@ name: Java CI with Maven
 
 on:
   push:
-    branches: [ master, main ]
+    branches: [main]
   pull_request:
-    branches: [ master, main ]
```

Comment:
do these work OK with an empty "pull_request:" container element in the yml?

Reply:
I got the impression from the documentation that the empty element is acceptable but I could be mistaken?
[This section](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-activity-types-and-filters-with-multiple-events) suggests that unconfigured events are valid?
[This stackoverflow page](https://stackoverflow.com/questions/64635032/github-actions-run-on-push-to-all-branches) also suggests that interpretation, though a wildcard glob may be a clearer alternative?